README.adoc

```diff
@@ -1,19 +1,25 @@
-= Mobile Docs Previewer
+= Mobile Docs Previewer (docs.aerogear.org)
 
-Allows you preview proposed changes to docs.aerogear.org without installing antora.
+Allows you preview a static html page with proposed changes to docs.aerogear.org without installing Antora.
 
 . Fork this repo.
-. Edit the site.yml to point to your fork and the branch you want to preview, for example:
+. Edit the site.yml to point to your forks of aerogear/mobile-docs and aerogear/mobile-security and specify the branchyou want to preview, for example:
 +
 ----
   - url: https://github.com/finp/mobile-docs.git
     branches: fix-things
+    
+  - url: https://github.com/finp/mobile-security.git
+    branches: fix-things
 ----
-. Commit and push your changes to master branch.
-. Create a netlify account if you don't already have one at https://www.netlify.com/
-. Create a *New site from Git* in netlify.
-. Select the fork you created in step one.
+. Edited `site.yml` must be in the `master` branch.
+. Create a link:https://www.netlify.com[^Netlify account] if you don't already have one.
+. Click *New site from Git* in Netlify.
+. Select the fork of `mobile-netlify` you created in step one.
 . Select to use the master branch.
 . Click *Deploy*.
+. First deploy can take up to 10 minutes. Subsequent deploys will take ~30 seconds.
```

Comment:
my experience is that subsequent deploys can also take many minutes

Reply:
40 seconds for removed content

Reply:
solved